processing/src/main/java/io/druid/query/extraction/TimeDimExtractionFn.java

```diff
@@ -74,6 +75,10 @@ public TimeDimExtractionFn(
   @Override
   public String apply(String dimValue)
```

Comment:
Could you add `@Nullable` annotations where appropriate? (including interface methods, and other impls)